jobs/kube-setup-scripts.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
this seems specific to data replication, maybe we should name this file something like `kube-setup-replication.sh`?